packages/rich-text/src/component/index.native.js

```diff
@@ -156,6 +165,16 @@ export class RichText extends Component {
 		return { ...value, start, end };
 	}
 
+	getHtmlForRecordCreation( value, tag ) {
+		if (
+			tag === 'p' &&
+			( ! value || value.length < 3 || ! value.startsWith( '<p>' ) )
```

Comment:
Why `value.length < 3` ?

Reply:
This was an over cautious check that startsWith would not go out of bounds, that I forgot to remove. Fixed here! 90f84421ec721a0382409fe0b3bd3c7026dcce7e